scripts/src/java/org/oppia/android/scripts/docs/KdocValidityCheck.kt

```diff
@@ -0,0 +1,347 @@
+package org.oppia.android.scripts.docs
+
+import org.jetbrains.kotlin.cli.common.CLIConfigurationKeys
+import org.jetbrains.kotlin.cli.common.messages.MessageCollector
+import org.jetbrains.kotlin.cli.jvm.compiler.EnvironmentConfigFiles
+import org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment
+import org.jetbrains.kotlin.com.intellij.openapi.util.Disposer
+import org.jetbrains.kotlin.com.intellij.psi.PsiManager
+import org.jetbrains.kotlin.com.intellij.testFramework.LightVirtualFile
+import org.jetbrains.kotlin.config.CompilerConfiguration
+import org.jetbrains.kotlin.idea.KotlinFileType
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.psi.KtClass
+import org.jetbrains.kotlin.psi.KtConstructorDelegationReferenceExpression
+import org.jetbrains.kotlin.psi.KtDeclaration
+import org.jetbrains.kotlin.psi.KtElement
+import org.jetbrains.kotlin.psi.KtFile
+import org.jetbrains.kotlin.psi.KtNamedFunction
+import org.jetbrains.kotlin.psi.KtObjectDeclaration
+import org.jetbrains.kotlin.psi.KtSecondaryConstructor
+import org.jetbrains.kotlin.psi.KtVariableDeclaration
+import org.oppia.android.scripts.common.RepositoryFile
+import org.oppia.android.scripts.proto.KdocValidityExemptions
+import java.io.File
+import java.io.FileInputStream
+
+/**
+ * Script for ensuring that KDocs validity on all non-private:
+ * - Classes
+ * - Functions
+ * - Values/fields
+ * - Explicit constructors
+ * - Companion objects
+ * - Nested classes
+ * - Enums
+ * - Annotations
+ * - Interfaces
+ *
+ * Note: If any of the above member has an override modifier present, then it is automatically
+ * exempted for the KDoc check.
+ *
+ * Usage:
+ *   bazel run //scripts:kdoc_validity_check -- <path_to_directory_root> <path_to_proto_binary>
+ *
+ * Arguments:
+ * - path_to_directory_root: directory path to the root of the Oppia Android repository.
+ * - path_to_proto_binary: relative path to the exemption .pb file.
+ *
+ * Example:
+ *   bazel run //scripts:kdoc_validity_check -- $(pwd) scripts/assets/kdoc_validity_exemptions.pb
+ */
+fun main(vararg args: String) {
+  // Path of the repo to be analyzed.
+  val repoPath = "${args[0]}/"
```

Comment:
use the same name given above in comment - `path_to_directory_root` in camel case

Reply:
Actually, we are using the same name "repoPath" for all the scripts till now, so will it be better to keep this the same in order to maintain consistency across all the scripts?